vendor/engines/sanger_sequencing/app/services/sanger_sequencing/sample_result_file_saver.rb

```diff
@@ -23,7 +23,9 @@ def save
                                    file_type: "sample_result", created_by: current_user.id,
                                    order_detail: sample.submission.order_detail,
                                    product_id: sample.submission.order_detail.product_id)
-      stored_file.save
+      success = stored_file.save
+      trigger_notification(stored_file) if success
+      success
```

Comment:
Might not be worth persuing but I'm wondering if these 3 lines could be reduced to `trigger_notification(stored_file) if stored_file.save`
If we can count on `trigger_notification` succeeding and returning something truthy…